conf/application.conf

```diff
@@ -14,6 +14,8 @@ mailgun {
   apiKey = "key-4b73f9bcf1a3b882d71262e679da5381" # this is a sandbox domain, OK to commit to git
 }
 
+emailDomain = "sandboxb581b83d5fc3420982d2af09f3b2f7df.mailgun.org"
```

Comment:
This domain no longer exists, so it might be confusing to write it here. I would just write `example.com` or something like that.